### PR TITLE
Rn entry 2021.06.2 (#2222)

### PR DESCRIPTION
**Pull request by brianduffield-okta**
## Description:
- **What's changed?** Added release notes for this week's release
- **Is this PR related to a Monolith release?** 2021.06.2

### Resolves:

* [OKTA-403264](https://oktainc.atlassian.net/browse/OKTA-403264)

### CODE REVIEW

File: packages/@okta/vuepress-site/docs/release-notes/2021/index.md [hunk 1]
```diff
@@ -3,6 +3,16 @@ title: Okta API Products Release Notes 2021
...
 
+### Weekly Release 2021.06.2
+
+| Change                                                                         | Expected in Preview Orgs |
+|--------------------------------------------------------------------------------|--------------------------|
+| [Bug fixed in 2021.06.2](#bug-fixed-in-2021-06-2)                              | June 16, 2021             |
+
+#### Bug fixed in 2021.06.2
+
+When the [features endpoint of the Apps API](/docs/reference/api/apps/#update-feature-for-application) was called to enable or disable user deactivation (`lifecycleDeactivate` property), the call didn’t toggle the `REACTIVATE_USERS` app feature. (OKTA-399233)
```
Comment (christinechoy-okta) [inline packages/@okta/vuepress-site/docs/release-notes/2021/index.md:14]:
```suggestion
When the [Features endpoint of the Apps API](/docs/reference/api/apps/#update-feature-for-application) was called to enable or disable user deactivation (`lifecycleDeactivate` property), the call didn’t toggle the `REACTIVATE_USERS` app feature. (OKTA-399233)
```

Comment (brianduffield-okta) [reply packages/@okta/vuepress-site/docs/release-notes/2021/index.md:14]: Thanks. Looks like it was capitalized in the API reference.

Comment (christinechoy-okta) [reply packages/@okta/vuepress-site/docs/release-notes/2021/index.md]: Not sure if just saying **(`lifecycleDeactivate` property)** is enough, or maybe it is. Maybe consider these options:
- (by using the `lifecycleDeactivate` property)
- (the `lifecycleDeactivate` property)

Comment (brianduffield-okta) [reply packages/@okta/vuepress-site/docs/release-notes/2021/index.md]: Ahhh! Found your comment. I committed too soon. As per Slack, will leave as is.

**Review by christinechoy-okta — APPROVED** [review]

**Pull request merged**